docs/DESIGN.md

```diff
@@ -0,0 +1,33 @@
+# Patchwork's Design Principles
+
+This document describes the design goals and philosophy of Patchwork. Features of and changes to Patchwork should always aim to follow these principles. If we really want to add a feature/make a change to Patchwork that goes against these principles, we should think about changing the principles first, rather than ignoring them and making the change anyway.
+
+This document should be version tracked, and any change to it should go through the [advice process](http://www.reinventingorganizationswiki.com/Decision_Making) with the stakeholders listed below.
+
+### Stakeholders
+* Patchwork contributors
+* Patchwork users
+* The broader Scuttlebutt community
+* People outside the Scuttlebutt community who might be affected
+
+## Design goals
+
+### Goals and scope: Patchwork should...
+* Follow the [Scuttlebutt principles](https://www.scuttlebutt.nz/principles/)
+* Be a client that people can feel confident recommending to their friends who aren't using Scuttlebutt
+  * Polished enough usability for people without patience or skills for tech troubleshooting
+  * Safety, blocking, harassment-stopping features for people concerned about those things
+  * Strike a balance: people should know how much data they're making public; also make it hard to find some public data which could be used for harassment
```

Comment:
This line feels like it should be moved/rephrased. Maybe put it in the previous line as "privacy" or something.